source/options.tsx

```diff
@@ -22,15 +22,18 @@ import isDevelopmentVersion from './helpers/is-development-version.js';
 import {doesBrowserActionOpenOptions} from './helpers/feature-utils.js';
 import {state as bisectState} from './helpers/bisect.js';
 
+type TokenType = 'classic' | 'fine_grained';
```

Comment:
Snake case is only used by GitHub's own permissions, but the rest of the form only uses camelCase. Let's stick to that here